### ISSUE TITLE
Unable to view stories due to "Stay Home" story feature

### ISSUE THREAD

**Issue by loopervfx** [MahStudio/Winsta#98]
The new "Stay Home" story feature doesn't appear correctly and also breaks the ability to view any stories from the Home view.

```
Arg_NullReferenceException 

Arg_NullReferenceException 
at UnofficialInstagram.View.MainView.<>c__DisplayClass15_0.b__1(Classes.Models.InstaReelFeed x) 
at UnofficialInstagram.View.MainView.StoriesLstV_ItemClick(Object sender, UI.Xaml.Controls.ItemClickEventArgs e) 
at UnofficialInstagram! 0x14a894e 
at UnofficialInstagram! 0x14fb996 
at UnofficialInstagram! 0x14fc14f 
at SharedLibrary! 0x4a0ad0 
at UnofficialInstagram! 0x13a9698 
at UnofficialInstagram! 0x13a95d4 
at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs) 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 
at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs) 
at System.EventHandler.Invoke(Object sender, Threading.Tasks.UnobservedTaskExceptionEventArgs e) 
at UnofficialInstagram! 0x14fb996 
at UnofficialInstagram! 0x14fc14f 

System manufacturer 
System Product Name 
WINDOWS 
```

**Comment by loopervfx** [comment]
A similar error occurs when viewing stories directly from a user's profile. It occurs **after the first story segment** has been viewed and for every subsequent story segment.
```
Arg_NullReferenceException

Arg_NullReferenceException
   at UnofficialInstagram.ViewModel.StoryHighlights.StoryViewVM.<>c__DisplayClass54_0.<MarkStoryAsSeen>b__0(Object x)
   at System.Func<System.Object,System.Boolean>.Invoke(Object arg)
   at SharedLibrary!<BaseAddress>+0x3c0251
   at UnofficialInstagram!<BaseAddress>+0x134f02f
   at UnofficialInstagram!<BaseAddress>+0x16cb18c
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at SharedLibrary!<BaseAddress>+0x65496e
   at SharedLibrary!<BaseAddress>+0x4f50d8
   at SharedLibrary!<BaseAddress>+0x4f543c
   at SharedLibrary!<BaseAddress>+0x4a0ad0
   at UnofficialInstagram!<BaseAddress>+0x13a9698
   at UnofficialInstagram!<BaseAddress>+0x13a95d4
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs)
   at System.EventHandler<System.Threading.Tasks.UnobservedTaskExceptionEventArgs>.Invoke(Object sender, Threading.Tasks.UnobservedTaskExceptionEventArgs e)
   at UnofficialInstagram!<BaseAddress>+0x14fb996
   at UnofficialInstagram!<BaseAddress>+0x14fc14f
<unknown>
System manufacturer
System Product Name
WINDOWS
```

**Comment by SirHaziel** [comment]
Similar error when trying to upload anything.
```
`Arg_NullReferenceException

Arg_NullReferenceException
   at UnofficialInstagram!<BaseAddress>+0x19a0c62
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at SharedLibrary!<BaseAddress>+0x65496e
   at SharedLibrary!<BaseAddress>+0x4f50d8
   at SharedLibrary!<BaseAddress>+0x4f543c
   at SharedLibrary!<BaseAddress>+0x4a0ad0
   at UnofficialInstagram!<BaseAddress>+0x13a9698
   at UnofficialInstagram!<BaseAddress>+0x13a95d4
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs)
   at System.EventHandler<System.Threading.Tasks.UnobservedTaskExceptionEventArgs>.Invoke(Object sender, Threading.Tasks.UnobservedTaskExceptionEventArgs e)
   at UnofficialInstagram!<BaseAddress>+0x14fb996
   at UnofficialInstagram!<BaseAddress>+0x14fc14f
<unknown>
System manufacturer
System Product Name
WINDOWS


DESKTOP-UN5T556
SKU`
```

**Comment by SirHaziel** [comment]
Tried to upload a new post after the update, still can't.
```
Arg_NullReferenceException

Arg_NullReferenceException
   at UnofficialInstagram!<BaseAddress>+0x19a0c62
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at SharedLibrary!<BaseAddress>+0x65496e
   at SharedLibrary!<BaseAddress>+0x4f50d8
   at SharedLibrary!<BaseAddress>+0x4f543c
   at SharedLibrary!<BaseAddress>+0x4a0ad0
   at UnofficialInstagram!<BaseAddress>+0x13a9698
   at UnofficialInstagram!<BaseAddress>+0x13a95d4
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object sender, ApplicationModel.Core.UnhandledErrorDetectedEventArgs eventArgs)
   at System.EventHandler<System.Threading.Tasks.UnobservedTaskExceptionEventArgs>.Invoke(Object sender, Threading.Tasks.UnobservedTaskExceptionEventArgs e)
   at UnofficialInstagram!<BaseAddress>+0x14fb996
   at UnofficialInstagram!<BaseAddress>+0x14fc14f
<unknown>
System manufacturer
System Product Name
WINDOWS


DESKTOP-UN5T556
SKU
```

**Comment by NGame1** [comment]
Fixed in V2.7.X updates.
